File: curso/src/main/java/br/ufmt/web/curso/repository/TarefaRepository.java
package br.ufmt.web.curso.repository;

import org.springframework.data.jpa.repository.support.JpaRepositoryImplementation;

import br.ufmt.web.curso.Tarefa;

public interface TarefaRepository 
  extends JpaRepositoryImplementation<Tarefa, Long> {
  
}

File: cursodotnet/Models/TarefaRepository.cs
namespace Br.Ufmt.Web.Curso.Repository
{
  public interface ITarefaRepository : IRepository<Tarefa, long>
  {  }

  public class TarefaRepository : Repository<Tarefa, long>, ITarefaRepository
  {
    public TarefaRepository(CursoContext context) : base(context)
    {
    }
  }
}